### PR TITLE
Add test for admin login feature

### PR DESCRIPTION
**Pull request by simrant27**
This PR adds e2e test setup files (Cucumber configuration) and adds tests for admin login feature.
``` feature
Scenario: login with valid credentials
Scenario: login with invalid credentials
```

### CODE REVIEW

File: tests/features/login.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    Scenario: login with valid credentials
+        When the admin logs in with following credentials
+            | email           | password    |
+            | admin@admin.com | admin123456 |
+
+        Then the admin should be navigated to the home page
+
+    Scenario Outline: login with invalid credentials
+        When the admin logs in with email "<invalidemail>" and password "<password>"
```
Comment (anon-pradip) [inline tests/features/login.feature:17]:
```suggestion
        When the admin logs in with email "<invalid-email>" and password "<password>"
```

File: tests/features/login.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        Then error message "<message>" should be shown
+        Examples:
+            | invalidemail     | password    | message          |
```
Comment (anon-pradip) [inline tests/features/login.feature:20]:
```suggestion
            | invalid-email     | password    | message          |
```

**Review by anon-pradip — CHANGES_REQUESTED** [review]

File: tests/features/login.feature [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    So that I can manage my products
+
+    Background:
+        Given the admin has navigated to the login page
+
+    Scenario: login with valid credentials
+        When the admin logs in with following credentials
+            | email           | password    |
+            | admin@admin.com | admin123456 |
+
```
Comment (prashant-gurung899) [inline tests/features/login.feature:13]:
```suggestion
```

File: tests/features/login.feature (hunk 1 above)
Comment (prashant-gurung899) [inline tests/features/login.feature:17]:
```suggestion
        When the admin tries to log in with email "<email>" and password "<password>"
```

**Comment by prashant-gurung899** [comment]
LGTM :+1:

**Pull request merged**